LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2004-2019 Fabien Potencier
+Copyright (c) 2004-2020 Fabien Potencier
```

Comment:
why not write `2004-present`?

(maybe question has already been asked, just to know :))
thank you

Reply:
I don't think you can claim copyright forever. That's probably in contradiction with copyright law and would make the clause void.

Reply:
Oki thx

Reply:
In fact, we can simply remove the end date. Copyright is claimed from a specific date up until "now", unless otherwise stated (i.e. if the copyright ended in 2017, it should be mentioned).

 See also https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated#2391555